todo_app.sql

```diff
@@ -0,0 +1,77 @@
+DROP DATABASE IF EXISTS todo_app;
+
+DROP USER IF EXISTS michael;
+
+CREATE USER michael ENCRYPTED PASSWORD '1234';
+
+CREATE DATABASE todo_app;
+
+\c todo_app;
+
+CREATE SEQUENCE table_name_id_seq;
```

Comment:
SEQUENCES do not need to be created manually. They will be made for you if you use the serial datatype.